### DISCUSSION TITLE
SSC inventory wiped

### DISCUSSION THREAD

**Discussion by SWright54** [Pryaxis/TShock#2007] [Q&A] [answered]
Hey,

Me and a friend just made a server so we could do a Master playthrough together. Basically we mostly just wanted TShock for SSC. We enabled it and it was working but all of a sudden we both lost our full inventories.

And now we can bring items in from singleplayer.

Nothing has changed as far as I know? Anyone have any ideas? Thanks!

**Comment by zaneshaw** [comment]
I have the same issue. I hope we get an answer because this set me back a week or two. #2005

**Comment by hakusaro** [comment]
Did you switch to superadmin temporarily @SWright54?

**Comment by hakusaro** [comment, marked as answer]
& can you post a screenshot of the startup? Can you post logs? Versions? it sounds like your SSC database is not in use, but not wiped?

**Reply by SWright54** [reply]
This seems to be happening on a server restart if that's anything to go by. Im guessing it's not saving to the database correctly?

**Reply by SWright54** [reply]
And neither chars were admins, no. I saw that on the documentation. 

Also using the latest version of TShock.

**Reply by SWright54** [reply]
2020-06-07 17:35:44 - CharacterManager: INFO: Skipping SSC Backup for MASTER BOBV1  <- I did find this in my logs though.

**Reply by SWright54** [reply]
tshock.ignore.*  <- I'm guessing it was this permission that was on guest permissions?
